config/logstash.yml

```diff
@@ -165,6 +165,13 @@
 # api.auth.basic.username: "logstash-user"
 # api.auth.basic.password: "s3cUreP4$$w0rD"
 #
+#
+# When configured with `api.auth.type` `basic`, you may set password policy mode.
+# This allows Logstash to WARN or ERROR when HTTP authentication password doesn't met the password policy requirements.
+# The default is WARN, setting to ERROR encourages using stronger password which is recommended.
+# The password policy requires non-empty minimum 5 char string to include digit, upper case letter and lower case letter.
+# api.auth.basic.password_policy.mode: WARN or ERROR
```

Comment:
```suggestion
# Applies when `api.auth.type` `basic`.
# Sets Logstash to WARN or ERROR when HTTP authentication password doesn't meet the password policy requirements.
# The default is WARN. Setting to ERROR enforces stronger passwords (recommended).
# The password policy requires non-empty minimum 5 char string that includes a digit, upper case letter and lower case letter.

# api.auth.basic.password_policy.mode: WARN or ERROR
```

Reply:
Thanks @karenzone for the review. Simple explanation looks great to me!
Could you please review the last update? We still want to keep the password policy controllable but not to expose in config file. I have added following lines say customers are able to control over policies. I also wonder if we need to add this statement to `settings_file.asciidoc`?!
```
# Set `api.auth.basic.password_policy.include.symbol` `REQUIRED` to require the password include special character.
# Set `api.auth.basic.password_policy.length.minimum` any number to extend 5 char length requirement.
```

Reply:
It seems to me like we should.  If users look at the doc and don't see these additional options, they'd have no reason to believe that there are closely related options that we're not telling them about and might even think that we're missing features they want.